frontend/src/components/NewsFeed.jsx

```diff
@@ -51,7 +51,7 @@ function NewsFeed({ communicationPlans }) {
               />
             );
           })}
-        </div>{" "}
+        </div>
         */
```

Comment:
pretty sure the `*/` should be removed as well ;)

Reply:
TY

Reply:
Its'ok